package.json

```diff
@@ -2,6 +2,7 @@
   "author": "Turo engineering",
   "description": "Turo semantic-release configuration",
   "dependencies": {
+    "@aensley/semantic-release-openapi": "1.2.1",
```

Comment:
This is the last version changed before `glob` is replaced by `fdir`, which is the source of our issues.

There is something about a difference of behavior in CI between the two libraries that breaks our toolchain as configured.